app/views/miq_policy/_form.html.haml

```diff
@@ -27,10 +27,10 @@
             = _("Applies To")
           .col-md-8
             - if @policy.id
-              = MiqPolicy::POLICY_TOWHAT_APPLIES_TO_CLASSES[@edit[:new][:towhat]]
+              = MiqPolicy.POLICY_TOWHAT_APPLIES_TO_CLASSES(@edit[:new][:towhat])
             - else
               = select_tag('towhat',
-                options_for_select(MiqPolicy::POLICY_TOWHAT_APPLIES_TO_CLASSES.invert, @edit[:new][:towhat]),
+                options_for_select(MiqPolicy.POLICY_TOWHAT_APPLIES_TO_CLASSES(nil).invert.map { |x| [_(x[0]), x[1]] }, @edit[:new][:towhat]),
```

Comment:
let's make the parameter default to nil on the other side, then you don't have to pass it here
```suggestion
                options_for_select(MiqPolicy.POLICY_TOWHAT_APPLIES_TO_CLASSES.invert.map { |x| [_(x[0]), x[1]] }, @edit[:new][:towhat]),
```